payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/CardNumberController.kt

```diff
@@ -31,12 +56,19 @@ internal class CardNumberController constructor(
     }
 
     private val _fieldState = combine(cardBrandFlow, _fieldValue) { brand, fieldValue ->
-        cardTextFieldConfig.determineState(brand, fieldValue)
+        cardTextFieldConfig.determineState(
+            brand,
+            fieldValue,
+            accountRange?.panLength ?: brand.getMaxLengthForCardNumber(fieldValue)
+        )
     }
     override val fieldState: Flow<TextFieldState> = _fieldState
 
     private val _hasFocus = MutableStateFlow(false)
 
+    override var accountRange: AccountRange? = null
+    override var accountRangeRepositoryJob: Job? = null
```

Comment:
Why are these overridden?  Can you just use the variables from the super class?

Reply:
The variables are in the interface which CardNumberController inherits from, so they're abstract and need to be instantiated here